docs/sdk/distribute/android.md

```diff
@@ -19,7 +19,7 @@ ms.tgt_pltfrm: android
 > * [iOS](ios.md)
 > * [Xamarin](xamarin.md)
 
-App Center Distribute will let your users install a new version of the app when you distribute it via App Center. With a new version of the app available, the SDK will present an update dialog to the users to either download or postpone the new version. Once they choose to update, the SDK will start to update your application. 
+App Center Distribute will let your users install a new version of the app when you distribute it via App Center. With a new version of the app available, the SDK will present an update dialog to the users to either download or postpone the new version. Once they choose to update, the SDK will start to update your application.
```

Comment:
We should start looking at removing passive voice from the docs. For example: When the SDK detects a new version of the app, the SDK presents an update dialog...